cli/src/pcluster/api/controllers/import_cdk.py

```diff
@@ -0,0 +1,27 @@
+import logging
```

Comment:
is there a reason why this file is in the controllers dir?

Reply:
Thank you for pointing this out. Moving to `templates`.